src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -837,15 +843,12 @@ private boolean isSnowflakeAuthenticator() {
         || AuthenticatorType.SNOWFLAKE.name().equalsIgnoreCase(authenticator);
   }
 
-  /**
-   * Returns true If authenticator is EXTERNALBROWSER.
-   *
-   * @return true if authenticator type is EXTERNALBROWSER
-   */
-  boolean isExternalbrowserAuthenticator() {
+  boolean isExternalbrowserOrOAuthFullFlowAuthenticator() {
```

Comment:
Full flow is a new name? WDYT about `isBrowserBasedAuthenticator`?

Reply:
CLIENT_CREDENTIALS flow is not browser based.

Reply:
Full flow as in opposed to OAUTH authenticator :)